itest/assertions.go

```diff
@@ -1210,7 +1210,7 @@ func AssertUniverseKeysEqual(t *testing.T, uniIDs []*unirpc.ID,
 }
 
 func AssertUniverseStats(t *testing.T, client unirpc.UniverseClient,
-	numProofs, numSyncs, numAssets int) {
+	numProofs, numSyncs, numTranches, numUnits int) {
```

Comment:
could add `numGroups` here? IIUC the count of grouped assets is fixed for the relevant tests.

Reply:
Makes sense, done.